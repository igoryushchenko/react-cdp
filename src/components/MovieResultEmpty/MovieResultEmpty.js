import React from 'react';

const MovieResultEmpty = () => (
    <div className="mx-auto">
      <h1>No films found</h1>
    </div>
);

export default MovieResultEmpty;
