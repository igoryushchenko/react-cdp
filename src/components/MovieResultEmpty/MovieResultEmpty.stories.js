import React from 'react';
import 'bootstrap/dist/css/bootstrap.min.css';
import { storiesOf } from '@storybook/react';
import MovieResultEmpty from './MovieResultEmpty';

storiesOf('MovieResultEmpty', module)
  .add('empty details', () => <MovieResultEmpty />);
