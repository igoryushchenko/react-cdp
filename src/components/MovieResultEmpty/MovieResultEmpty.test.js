import React from 'react';
import { shallow } from 'enzyme';
import MovieResultEmpty from './MovieResultEmpty';

describe('MovieResultEmpty component', () => {
  it('should be render correctly', () => {
    const component = shallow(<MovieResultEmpty />);
    expect(component).toMatchSnapshot();

    expect(component.text()).toEqual('No films found');
  });
});
