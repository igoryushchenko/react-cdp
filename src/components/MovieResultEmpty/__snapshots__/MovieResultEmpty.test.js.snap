// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieResultEmpty component should be render correctly 1`] = `ShallowWrapper {}`;
